providers/apns/src/lib/apns.provider.ts

```diff
@@ -44,7 +44,7 @@ export class APNSPushProvider implements IPushProvider {
     });
     const res = await this.provider.send(notification, options.target);
 
-    if (res.failed) {
+    if (res.failed.length > 0) {
```

Comment:
Reference for this change:
https://github.com/node-apn/node-apn/blob/master/doc/provider.markdown#failed